src/Twig/ArrayExtension.php

```diff
@@ -139,9 +135,6 @@ private function getSortOrder($name = '-datepublish'): array
 
     /**
      * Helper function for sorting an array of \Bolt\Legacy\Content.
-     *
-     * @param \Bolt\Legacy\Content|array $a
-     * @param \Bolt\Legacy\Content|array $b
      */
     private function orderHelper($a, $b): bool
```

Comment:
array $a, array $b

Reply:
This is not yet ported to v4, but $a and $b can be either `array` or an instance of `Content`, so that's why i didn't typehint them. In advance of the functionality gets ported over, I will lock it down to `Content`. :-)

Reply:
So let's leave previous phpdoc but update Legacy/Content to Content and add @todo that Content must implement Traversable or refactor is needed.